hawkular-tenant-jaxrs-filter/pom.xml

```diff
@@ -64,8 +64,8 @@
 
   <dependencies>
     <dependency>
-      <groupId>javax</groupId>
-      <artifactId>javaee-api</artifactId>
+      <groupId>org.jboss.resteasy</groupId>
```

Comment:
They already depend on resteasy so I really do hope we can all standardize on using tenant filter.